package-lock.json

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
so far, we chose not to commit the `package-lock.json`, but I know this is an eternal debate. You have more experience with this, so if you feel strongly that we should version it, we'll defer to your judgment

Reply:
it helps with consistency across deployments and environments, but I don't have a strong preference